### YamlMime:ManagedReference
items:
- uid: Innovt.Cloud.AWS.Cognito.Exceptions
  commentId: N:Innovt.Cloud.AWS.Cognito.Exceptions
  id: Innovt.Cloud.AWS.Cognito.Exceptions
  children:
  - Innovt.Cloud.AWS.Cognito.Exceptions.CodeMismatchException
  - Innovt.Cloud.AWS.Cognito.Exceptions.ExpiredCodeException
  - Innovt.Cloud.AWS.Cognito.Exceptions.InternalException
  - Innovt.Cloud.AWS.Cognito.Exceptions.InvalidPasswordException
  - Innovt.Cloud.AWS.Cognito.Exceptions.PasswordResetRequiredException
  - Innovt.Cloud.AWS.Cognito.Exceptions.UserNameAlreadyExistsException
  - Innovt.Cloud.AWS.Cognito.Exceptions.UserNotAuthorizedException
  - Innovt.Cloud.AWS.Cognito.Exceptions.UserNotConfirmedException
  - Innovt.Cloud.AWS.Cognito.Exceptions.UserNotFoundException
  langs:
  - csharp
  - vb
  name: Innovt.Cloud.AWS.Cognito.Exceptions
  nameWithType: Innovt.Cloud.AWS.Cognito.Exceptions
  fullName: Innovt.Cloud.AWS.Cognito.Exceptions
  type: Namespace
  assemblies:
  - Innovt.Cloud.AWS.Cognito
references:
- uid: Innovt.Cloud.AWS.Cognito.Exceptions.CodeMismatchException
  commentId: T:Innovt.Cloud.AWS.Cognito.Exceptions.CodeMismatchException
  href: Innovt.Cloud.AWS.Cognito.Exceptions.CodeMismatchException.html
  name: CodeMismatchException
  nameWithType: CodeMismatchException
  fullName: Innovt.Cloud.AWS.Cognito.Exceptions.CodeMismatchException
- uid: Innovt.Cloud.AWS.Cognito.Exceptions.ExpiredCodeException
  commentId: T:Innovt.Cloud.AWS.Cognito.Exceptions.ExpiredCodeException
  href: Innovt.Cloud.AWS.Cognito.Exceptions.ExpiredCodeException.html
  name: ExpiredCodeException
  nameWithType: ExpiredCodeException
  fullName: Innovt.Cloud.AWS.Cognito.Exceptions.ExpiredCodeException
- uid: Innovt.Cloud.AWS.Cognito.Exceptions.InternalException
  commentId: T:Innovt.Cloud.AWS.Cognito.Exceptions.InternalException
  href: Innovt.Cloud.AWS.Cognito.Exceptions.InternalException.html
  name: InternalException
  nameWithType: InternalException
  fullName: Innovt.Cloud.AWS.Cognito.Exceptions.InternalException
- uid: Innovt.Cloud.AWS.Cognito.Exceptions.InvalidPasswordException
  commentId: T:Innovt.Cloud.AWS.Cognito.Exceptions.InvalidPasswordException
  href: Innovt.Cloud.AWS.Cognito.Exceptions.InvalidPasswordException.html
  name: InvalidPasswordException
  nameWithType: InvalidPasswordException
  fullName: Innovt.Cloud.AWS.Cognito.Exceptions.InvalidPasswordException
- uid: Innovt.Cloud.AWS.Cognito.Exceptions.PasswordResetRequiredException
  commentId: T:Innovt.Cloud.AWS.Cognito.Exceptions.PasswordResetRequiredException
  href: Innovt.Cloud.AWS.Cognito.Exceptions.PasswordResetRequiredException.html
  name: PasswordResetRequiredException
  nameWithType: PasswordResetRequiredException
  fullName: Innovt.Cloud.AWS.Cognito.Exceptions.PasswordResetRequiredException
- uid: Innovt.Cloud.AWS.Cognito.Exceptions.UserNameAlreadyExistsException
  commentId: T:Innovt.Cloud.AWS.Cognito.Exceptions.UserNameAlreadyExistsException
  href: Innovt.Cloud.AWS.Cognito.Exceptions.UserNameAlreadyExistsException.html
  name: UserNameAlreadyExistsException
  nameWithType: UserNameAlreadyExistsException
  fullName: Innovt.Cloud.AWS.Cognito.Exceptions.UserNameAlreadyExistsException
- uid: Innovt.Cloud.AWS.Cognito.Exceptions.UserNotAuthorizedException
  commentId: T:Innovt.Cloud.AWS.Cognito.Exceptions.UserNotAuthorizedException
  href: Innovt.Cloud.AWS.Cognito.Exceptions.UserNotAuthorizedException.html
  name: UserNotAuthorizedException
  nameWithType: UserNotAuthorizedException
  fullName: Innovt.Cloud.AWS.Cognito.Exceptions.UserNotAuthorizedException
- uid: Innovt.Cloud.AWS.Cognito.Exceptions.UserNotConfirmedException
  commentId: T:Innovt.Cloud.AWS.Cognito.Exceptions.UserNotConfirmedException
  href: Innovt.Cloud.AWS.Cognito.Exceptions.UserNotConfirmedException.html
  name: UserNotConfirmedException
  nameWithType: UserNotConfirmedException
  fullName: Innovt.Cloud.AWS.Cognito.Exceptions.UserNotConfirmedException
- uid: Innovt.Cloud.AWS.Cognito.Exceptions.UserNotFoundException
  commentId: T:Innovt.Cloud.AWS.Cognito.Exceptions.UserNotFoundException
  href: Innovt.Cloud.AWS.Cognito.Exceptions.UserNotFoundException.html
  name: UserNotFoundException
  nameWithType: UserNotFoundException
  fullName: Innovt.Cloud.AWS.Cognito.Exceptions.UserNotFoundException
